### ISSUE TITLE
Investigate Machine Learning Integration for color contrast

### ISSUE THREAD

**Issue by xNS5** [xNS5/Axe-Core-Enhancements#26]
Currently the aXe API doesn't have a means to detect color contrast and labels it as something that requires a human to verify. I recently discovered that AWS has API for OCR -- which will not only identify text on an image but also return the coordinates on the page.

Frankly I think it might be a bit of a pain to cover cases where each letter is a different color, but the easiest thing to do is just compare the color of the text at indexes [x,y] and compare it to the background color. 

https://docs.aws.amazon.com/rekognition/latest/dg/text-detecting-text-procedure.html
